### PR TITLE
Tear down rummager in all environments (AWS and Carrenza)

### PR DESCRIPTION
**Pull request by barrucadu**
This should remove the rummager app and all of its related components (rabbitmq stuff, etc).  When we're happy that that has worked, the rummager hieradata and puppet can be removed.

---

[Trello card](https://trello.com/c/nABGyw0B/653-%F0%9F%92%80-remove-rummager-%F0%9F%92%80)

### CODE REVIEW

File: hieradata/common.yaml [hunk 1]
```diff
@@ -562,6 +558,9 @@ govuk::apps::local_links_manager::unicorn_worker_processes: "4"
 
 govuk::apps::mapit::enabled: true
 
+govuk::apps::rummager::enable_rummager: false
+govuk::apps::search_api::rummager_enable: false
```
Comment (cbaines) [inline hieradata/common.yaml]: I'm finding these changes a little hard to follow... Will setting this here have any effect, given the change to the `search` node class in this same file?

Comment (cbaines) [reply hieradata/common.yaml]: (this applying to both lines I guess, particularly the first one)

Comment (barrucadu) [reply hieradata/common.yaml]: I think I should probably leave the `node_class` how it is for now: https://github.com/alphagov/govuk-puppet/blob/3e0492ca60ff066db99a02140f38e3729a211533/modules/govuk/manifests/node/s_base.pp#L46-L53
I'll update the PR
